src/Bootstrap/src/Autoconfig/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,245 @@
+                if (assemblies.TryGetValue(name, out assembly))
+                {
+                    return assembly;
+                }
+
+                if (args.Name?.Contains(".resources") ?? false)
+                {
+                    return args.RequestingAssembly;
+                }
+
+                missingAssemblies.Add(name); // throw it in there to prevent recursive attempts to resolve
+                assembly = Assembly.Load(name);
+                missingAssemblies.Remove(name);
+                return assembly;
+            }
+        }
+
+        public static IHostBuilder AddSteeltoe(this IHostBuilder hostBuilder, ILoggerFactory loggerFactory = null)
+        {
+            var logger = loggerFactory?.CreateLogger(LoggerName) ?? NullLogger.Instance;
+            hostBuilder.Properties[LoggerName] = logger;
+
+            if (!hostBuilder.WireIfLoaded(WireConfigServer, SteeltoeAssemblies.Steeltoe_Extensions_Configuration_ConfigServerCore))
+            {
+                hostBuilder.WireIfLoaded(WireCloudFoundryConfiguration, SteeltoeAssemblies.Steeltoe_Extensions_Configuration_CloudFoundryCore);
+            }
+
+            if (IsAssemblyLoaded(SteeltoeAssemblies.Steeltoe_Extensions_Configuration_KubernetesCore) && IsRunningInKubernetes())
+            {
+                WireKubernetesConfiguration(hostBuilder);
+            }
+
+            hostBuilder.WireIfLoaded(WirePlaceholderResolver, SteeltoeAssemblies.Steeltoe_Extensions_Configuration_PlaceholderCore);
+
+            if (IsAssemblyLoaded(SteeltoeAssemblies.Steeltoe_Connector_ConnectorCore))
+            {
+                hostBuilder.WireIfAnyLoaded(WireMySqlConnection, MySqlTypeLocator.Assemblies);
+                hostBuilder.WireIfAnyLoaded(WireMongoClient, MongoDbTypeLocator.Assemblies);
+                hostBuilder.WireIfAnyLoaded(WireOracleConnection, OracleTypeLocator.Assemblies);
+                hostBuilder.WireIfAnyLoaded(WirePostgresConnection, PostgreSqlTypeLocator.Assemblies);
+                hostBuilder.WireIfAnyLoaded(WireRabbitMqConnection, RabbitMQTypeLocator.Assemblies);
+                hostBuilder.WireIfAnyLoaded(WireRedisConnectionMultiplexer, RedisTypeLocator.StackExchangeAssemblies);
+                hostBuilder.WireIfAnyLoaded(WireDistributedRedisCache, RedisTypeLocator.MicrosoftAssemblies.Except(new[] { "Microsoft.Extensions.Caching.Abstractions" }).ToArray());
+                hostBuilder.WireIfAnyLoaded(WireSqlServerConnection, SqlServerTypeLocator.Assemblies);
+            }
+
+            hostBuilder.WireIfLoaded(WireKubernetesActuators, SteeltoeAssemblies.Steeltoe_Management_KubernetesCore);
+            hostBuilder.WireIfLoaded(WireDiscoveryClient, SteeltoeAssemblies.Steeltoe_Discovery_ClientCore);
+            hostBuilder.WireIfLoaded(WireCloudFoundryActuators, SteeltoeAssemblies.Steeltoe_Management_CloudFoundryCore);
```

Comment:
`Management.CloudFoundryCore` isn't needed anymore, it's all obsolete and all functionality is contained in EndpointCore

Reply:
Probably fine to leave this in for anybody that is still using it... we can drop it if/when the package is completely removed